src/Listeners/CreateParticipant.php

```diff
@@ -27,7 +27,7 @@ public function handle(WaiverSigned $event)
     {
         $signature = $event->signature;
 
-        $participant = app('participant')->withTrashed()->firstOrNew(['email' => $signature->email]);
+        $participant = app('participant')->withTrashed()->firstOrNew(['email' => $signature->email()->email]);
```

Comment:
This will rely on a new release of tipoff/bookings with the same email address. It's a lot of interdependency, so maybe it would be best to create a function on the Participant class that returns the email address as a string.